puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -9,6 +9,8 @@
   $backend_glusterfs_name = 'glusterfs',
   $backend_iscsi          = false,
   $backend_iscsi_name     = 'iscsi',
+  $backend_netapp         = false,
```

Comment:
@jguiditta thanks for taking a look through this.  I'm not entirely familiar with astapor, so help me understand what your comment above means.  I have added parameters above to `puppet/modules/quickstack/manifests/cinder_volume.pp`, which seems to be the necessary changes to `quickstack::cinder`.  Are you suggesting that I remove these parameters or add them to a different file?  Thanks!